napari/layers/labels/labels.py

```diff
@@ -649,6 +664,17 @@ def _raw_to_displayed(self, raw):
         else:
             raise ValueError("Unsupported Color Mode")
 
+        if self.contour:
+            image = np.zeros_like(raw)
+            struct_elem = ndi.generate_binary_structure(raw.ndim, 1)
+            boundaries = ndi.grey_dilation(
+                raw, footprint=struct_elem
+            ) != ndi.grey_erosion(raw, footprint=struct_elem)
+            image[boundaries] = raw[boundaries]
+            image = np.where(
+                raw > 0, low_discrepancy_image(image, self._seed), 0
```

Comment:
As discovered by @sofroniewn, this should be:

```python
                 image > 0, low_discrepancy_image(image, self._seed), 0
```

not

```python
                 raw > 0, low_discrepancy_image(image, self._seed), 0
```

Reply:
Oops, I can make a follow up PR to fix this.

Reply:
> Oops, I can make a follow up PR to fix this.

Ah @kir0ul please see #2296, which also adds contour thickness as an int

Reply:
Ah already done then.